pkg/pxc/app/statefulset/node.go

```diff
@@ -265,6 +266,75 @@ func (c *Node) AppContainer(spec *api.PodSpec, secrets string, cr *api.PerconaXt
 		}...)
 	}
 
+	if cr.CompareVersionWith("1.17.0") >= 0 {
```

Comment:
We can't include it in 1.17.0 :( let's do it for 1.18.0. Thanks.

Reply:
done